apps/codeforafrica/src/payload/blocks/PostsOverView.js

```diff
@@ -0,0 +1,66 @@
+import linkGroup from "../fields/links/linkGroup";
+
+const primaryTags = [
+  {
+    label: "Stories",
+    value: "stories",
+  },
+  {
+    label: "Opportunities",
+    value: "opportunities",
+  },
+];
+
+const PostsOverview = {
+  slug: "posts-overview",
+  imageURL: "/images/cms/blocks/posts_overview.jpg",
+  imageAltText: "Stories & Opportunities Overview",
+  fields: [
+    {
+      name: "title",
+      label: "Title",
+      required: true,
+      type: "text",
+    },
+    {
+      name: "primaryTag",
+      label: "Post Type",
+      required: true,
+      type: "select",
+      options: primaryTags,
+      hasMany: false,
+    },
+    {
+      name: "featured",
+      type: "relationship",
+      relationTo: "posts",
+      hasMany: false,
+      required: true,
+    },
+    {
+      name: "posts",
+      type: "relationship",
+      relationTo: "posts",
+      hasMany: true,
+      minRows: 3,
+      maxRows: 3,
```

Comment:
These needs to be posts of type stories only.

You can also make this a group as well
```js
stories: {
  items: // This is the relationship field
  action: // This is the action linkGroup field
}
```